app/src/main/res/values/strings.xml

```diff
@@ -461,6 +461,7 @@
   <string name="correct_submitted_answer_with_append">Correct submitted answer: %s</string>
   <string name="incorrect_submitted_answer">Incorrect submitted answer</string>
   <string name="incorrect_submitted_answer_with_append">Incorrect submitted answer: %s</string>
+  <string name="splash_activity_title">Splash Activity</string>
```

Comment:
Instead of `Splash Activity` this should be `Oppia`. @BenHenning Any thoughts on this one?

Reply:
Yes, 'Oppia' sounds correct. We should use the existing untranslated string for that purpose (https://github.com/oppia/oppia-android/blob/develop/app/src/main/res/values/untranslated_strings.xml#L3).